memcached/init.lua

```diff
@@ -217,6 +217,7 @@ local function config_initial(cfg)
     box.error{ reason = err }
 end
 
+-- luacheck: no unused
```

Comment:
If I'll remove it:

```
Checking memcached/init.lua                       1 warning

    memcached/init.lua:220:16: unused function config_help
```

But the function is actually unused. We should either drop it or expose (it looks useful). Or, well, at least add a TODO mark about exposing.

Reply:
Added a comment.

Reply:
Nice!